src/frontend/context/BehandlingContext.ts

```diff
@@ -13,10 +15,14 @@ export const [BehandlingProvider, useBehandling] = constate(
     ({ behandling, hentBehandling }: Props) => {
         const behandlingErRedigerbar = erBehandlingRedigerbar(behandling);
 
+        const [statusPåVentRedigering, settStatusPåVentRedigering] = useState(false);
```

Comment:
Kan ikke denne staten bare ligge i `BehandlingInnhold` og så sendes som props til `SettPåVentContainer`?

Jeg synes det er fint å holde useBehandling litt ryddig, og det er vel ikke andre komponenter som har bruk for denne staten?

Reply:
Men ser jo at den kanskje må ligge her hvis vi skal flytte det ut til en hamburgermeny et annet sted 🤔

Reply:
Den kan jo flyttes hit om det faktisk er behov for å flytte den hit eventuelt. Marie hadde jo nevnt at den visuelt bør høre til en behandling, så sannsynligheten for at den havner utenfor behandlingscontext er ikke så stor

Reply:
Ulempen er at det blir mer støy i koden sånn ellers.
https://github.com/navikt/tilleggsstonader-sak-frontend/pull/221/commits/f5eb7577c7ebdd8ada37caa1597bd20a4c296199